x/gov/handler.go

```diff
@@ -114,7 +114,7 @@ func EndBlocker(ctx sdk.Context, keeper Keeper) (resTags sdk.Tags) {
 		resTags.AppendTag(tags.Action, tags.ActionProposalDropped)
 		resTags.AppendTag(tags.ProposalID, proposalIDBytes)
 
-		logger.Info(fmt.Sprintf("Proposal %d - \"%s\" - didn't mean minimum deposit (had only %s), deleted",
+		logger.Info(fmt.Sprintf("Proposal %d (%s) didn't meet minimum deposit (had only %s). Deleted",
```

Comment:
`had only %s` of what? I think we can add something here for clarification 👍

Reply:
not sure if we have a standard on this, but I don't see the use of capitalizing. This reads better to me:

```
"proposal %d (%s) didn't meet minimum deposit (had only %s); deleted"
```

This can apply for the rest of the changes.

Reply:
Not capitalizing is standard for go errors, not sure about log info messages.

Reply:
correct -- I think we should follow the same convention here.